docs/user/handle-attachments.md

```diff
@@ -95,12 +95,12 @@ for the corresponding definition yourself and delete it from the array.
 The following example shows how to add an attachment to a PDF/A-3B compliant document
 without breaking compliance:
 
-```python
+```{testcode}
 from pypdf import PdfWriter
 from pypdf.constants import AFRelationship
 from pypdf.generic import create_string_object, ArrayObject, NameObject
 
-writer = PdfWriter(clone_from="pdf_a3b.pdf")
+writer = PdfWriter(clone_from="example.pdf")
```

Comment:
The example file is most likely not compliant to PDF/A-3B, thus this example would fail validation on the user side.

Reply:
True. But this is the same problem as 

> Code snippets now executable. But it make sense to review all used input files and make sure that content match text and images used in documentation. It is not always the case. I consider it as *out of scope* of this task.

I think after merging this PR, all code snippets need to be revisited to make sure that output pdfs match exactly what documentation describes.